charts/tfy-prometheus-config/templates/prometheusRules/alerting-rules.yaml

```diff
@@ -4,7 +4,7 @@ kind: PrometheusRule
 metadata:
   name: {{ .Values.prometheusRules.containerRules.name }}
   labels:
-    {{- include "prometheusRules.labels" . | nindent 4 }}
+    {{- include "alerting-rules.labels" . | nindent 4 }}
```

Comment:
How having alerting-rules.labels better then a single prometheus.Labels. We should add containerRule labels also in the template function

Reply:
I saw that you are adding that in the helpers template. Just change the name here.